pkg/build/builder/cmd/builder.go

```diff
@@ -7,29 +7,27 @@ import (
 	"os"
 	"path/filepath"
 
-	istorage "github.com/containers/image/storage"
-	"github.com/containers/image/types"
-	"github.com/containers/storage"
-	realglog "github.com/golang/glog"
```

Comment:
do you understand the implications of this change?  There's a reason we were distinguishing between the glog package and the glog variable before...

Reply:
This was meant to clean up the imports.

Reply:
My understanding is that this is related to how we handle buildah logs. The `glog` variable is a wrapper which lets us override the effective log level from buildah.

Reply:
no, the loglevel in buildah is handled elsewhere.  the glog wrapper is, if i recall correctly, used to avoid printing timestamps unless debug level logging is enabled.

Anyway based on your comments it makes sense now that it's going away here.